bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/builder/ProcessBuilder.java

```diff
@@ -89,11 +89,20 @@ public IntermediateCatchEventBuilder linkCatchEvent(final String id) {
   protected void setCoordinates(final BpmnShape targetBpmnShape) {
     final Bounds bounds = targetBpmnShape.getBounds();
     bounds.setX(100);
-    bounds.setY(100);
+    bounds.setY(100 + getLowestHeight());
```

Comment:
:x: The `CoordinatesGenerationTest` doesn't like your changes. 

This test fails because the coordinated doesn't match the expectations anymore.